README.md

```diff
@@ -3,7 +3,11 @@ React Frontend for giftaid submissions.
 
 ## Installation
 
+You will need to be a member of the Comic Relief organisation on NPM in order to install this project, as it has the
+private NPM package `@comicrelief/data-models` as a dependency.
```

Comment:
Minor, but maybe say "dev dependency" here?

Reply:
It's a breed of dependency 

> I think `test-utils` has `data-models` as a peer dependency because only some of the test utils (specifically, ERP and User Service testers) require the data models. Peer dependencies are optional things that you might not always need; if you're not using those particular testers, you don't need to install `data-models`. However this repo does use `ErpTester`, so it needs `data-models` installed. Hope that makes sense!

Sure does, thanks

Reply:
> Minor, but maybe say "dev dependency" here?

Will update